model/src/main/proto/exploration.proto

```diff
@@ -85,7 +85,50 @@ message Interaction {
   Outcome default_outcome = 6;
   // Mapping from the name of a customization arg to the default interaction
   // object of the customization arg
-  map<string, InteractionObject> customization_args = 7;
+  map<string, SchemaObject> customization_args = 7;
+}
+
+// Structure for schema objects. See https://github.com/oppia/oppia/blob/fcb44f8/schema_utils.py#L50 for context.
+message SchemaObject {
+  oneof object_type {
+    bool bool_value = 1;
+    CustomSchemaValue custom_schema_value = 2;
+    DictObject dict_value = 3;
+    float float_value = 4;
+    string normalized_string = 5;
+    int32 signed_int = 6;
+    SchemaObjectList schema_object_list = 7;
+    string unicode_value = 8;
+    SubtitledHtml subtitled_html = 9;
+    SubtitledUnicode subtitled_unicode = 10;
+    double real = 11;
+  }
+}
+
+// Structure for a dict object
+message DictObject {
+  map<string, SchemaObject> dict_value = 1;
+}
+
+// Structure for a list of schema object
+message SchemaObjectList {
+  repeated SchemaObject schema_object = 1;
+}
+
+// Structure which supports custom schemas
+message CustomSchemaValue {
+  oneof custom_type {
+    ImageWithRegions image_with_regions = 1;
+    SubtitledHtml subtitled_html = 2;
+    SubtitledUnicode subtitled_unicode = 3;
+    ClickOnImage click_on_image = 4;
+  }
+}
+
+// Structure for unicode and content id.
+message SubtitledUnicode {
```

Comment:
Doesn't seem used anywhere above--please remove.

Reply:
Removed this for now but might need to introduce this back in future keeping https://github.com/oppia/oppia/blob/fcb44f8/schema_utils.py#L50 in mind.

Resolving this on behalf of Ben.